backend/src/main/java/heartbeat/service/board/jira/JiraService.java

```diff
@@ -112,22 +109,24 @@ public BoardConfigDTO getJiraConfiguration(BoardType boardType, BoardRequestPara
 		try {
 			JiraBoardConfigDTO jiraBoardConfigDTO = getJiraBoardConfig(baseUrl, boardRequestParam.getBoardId(),
 					boardRequestParam.getToken());
-			CompletableFuture<List<TargetField>> targetFieldFuture = getTargetFieldAsync(baseUrl, boardRequestParam);
-			CompletableFuture<List<TargetField>> ignoredTargetFieldFuture = getIgnoredTargetFieldAsync(baseUrl,
-					boardRequestParam);
 			CompletableFuture<JiraColumnResult> jiraColumnsFuture = getJiraColumnsAsync(boardRequestParam, baseUrl,
 					jiraBoardConfigDTO);
-
-			return jiraColumnsFuture.thenCombine(targetFieldFuture,
-					(jiraColumnResult, targetFields) -> getUserAsync(boardType, baseUrl, boardRequestParam)
-						.thenApply(users -> BoardConfigDTO.builder()
-							.targetFields(targetFields)
-							.jiraColumnResponse(jiraColumnResult.getJiraColumnResponse())
-							.ignoredTargetFields(ignoredTargetFieldFuture.join())
-							.users(users)
-							.build())
-						.join())
-				.join();
+			CompletableFuture<List<TargetField>> targetFieldFuture = getTargetFieldAsync(baseUrl, boardRequestParam);
+			List<TargetField> ignoredTargetFields = getTargetFieldAsync(baseUrl, boardRequestParam).join()
```

Comment:
In line 114 it have already got the target fields. Why do you need to call the getTargetFieldAsync again to get target fields?